packages/contracts/scripts/osx-versions-aliases.ts

```diff
@@ -5,4 +5,7 @@
  *
  * To add a new version alias, append it to this array.
  */
-export const OSX_VERSION_ALIASES = ['@aragon/osx-v1.0.1/'];
+export const OSX_VERSION_ALIASES = [
+  '@aragon/osx-v1.0.1/',
+  '@aragon/osx-v1.3.0-rc0.2/',
```

Comment:
We must remember to bump this to the final release later on
Still not having a release v1.3.0 is becoming a problem.

Reply:
is it part of the deployment/update checklist? should it be ?